sdks/python/apache_beam/examples/fastavro_it_test.py

```diff
@@ -45,12 +45,15 @@
 from __future__ import absolute_import
 from __future__ import division
 
+import json
 import logging
 import os
 import sys
 import unittest
 import uuid
 
+# pylint: disable=wrong-import-order, wrong-import-position
```

Comment:
Does this need to be moved up here? If possible, I would limit the scope to the try/except block as it used to be.